Roadmap.md

```diff
@@ -1,188 +1,235 @@
+- PRs: -
 
-It’s been an interesting year for npm clients. Once unrivaled as the tool of choice for installing npm packages, the `npm` command-line tool faced some serious competition starting last September from an innovative tool called `yarn`, which promised fast, reproducible installs based on `yarn.lock` files.
+## Recently completed
 
-The popularity of `yarn` led Meteor to support `meteor yarn` in addition to `meteor npm` (though you had to `meteor npm install --global yarn` first, so `npm` still had an advantage). Our own  Galaxy Server and Optics apps, which are built with Meteor, switched over to `yarn` soon after its release. The appeal was undeniable.
+### Different JS bundles for modern versus legacy browsers
 
-This competition was a good thing for JavaScript developers, first because yarn solved some long-standing problems with `npm`, and later because `npm@5` responded by shipping its own implementation of some similar features, with `package-lock.json` files and automatic addition of npm install-ed packages to `package.json`.
+- Status: shipped in Meteor 1.6.2.
+- PRs: https://github.com/meteor/meteor/pull/9439
 
-These improvements to `npm` will benefit all Meteor developers, even those who keep using `yarn`, because package dependencies specified with `Npm.depends` are automatically installed using `npm`, and `npm@5` performs those installations much faster and more reliably.
-
-Meteor is careful to remain agnostic about how you choose to populate your `node_modules` directories, so we fully expect that `meteor npm` and `meteor yarn` will remain equally good alternatives for that purpose.
-
-## Dynamic `import(...)`
-
-*Status: Shipped in 1.5*
-
-The banner feature of this effort will be first-class support for [dynamic `import(...)`](https://github.com/tc39/proposal-dynamic-import), which enables asynchronous module fetching.
-
-Read the recent [blog post](https://blog.meteor.com/meteor-1-5-react-loadable-f029a320e59c) for an overview of how this system will work in Meteor 1.5.
-
-Remaining work can be found [here](https://github.com/meteor/meteor/blob/release-1.5/packages/dynamic-import/TODO.md), though not all of those ideas will necessarily block the initial 1.5 release.
-
-
-## Rebuild performance improvements
-
-*Status: Shipped in 1.4.2*
-
-Rebuild performance refers to the length of time between changing a file in development and being able to reload your app in a browser. After extensive profiling to identify performance hot-spots, and with careful caching of previously completed work, Meteor 1.4.2 takes substantially less time to rebuild most apps, especially larger apps.
+### Eliminate the need for an `imports` directory
 
+- Status: shipped in Meteor 1.6.2.
+- PRs: https://github.com/meteor/meteor/pull/9690, https://github.com/meteor/meteor/pull/9714, https://github.com/meteor/meteor/pull/9715
 
-## MongoDB updates
+### Make Mongo more optional
 
-*Status: Shipped in 1.4*
+- Status: shipped in Meteor 1.6.2.
```

Comment:
Not really. One of big selling points of Meteor is drop-in accounts. I think those should also be made so that database connection can be swapped to different database backends.

Reply:
I have a few items that I would like to add to the roadmap in the next cycle, one of them is the accounts system independence

- Improve accounts system
- HTTP/2 Websocket Support
- Improving MongoDB Oplog Tailing

If someone is willing to take the lead now we could include already.